kratos/python_scripts/assign_vector_by_direction_to_element_process.py

```diff
@@ -46,10 +46,14 @@ def __init__(self, Model, settings ):
         if settings.Has("modulus"):
             if settings["modulus"].IsString():
                 default_settings["modulus"].SetString("0.0")
+        else:
+            raise RuntimeError("Please specify the modulus of the vector")
 
         if settings.Has("direction"):
             if settings["direction"].IsString():
                 default_settings["direction"].SetString("Automatic")
+        else:
+            raise RuntimeError("Please specify the direction of the vector")
```

Comment:
not specifying seems to be ok here no?

Reply:
I guess it doesn't really matter since the default of [0,0,0] will cause an exception later on anyways (when attempting to normalize it in `assign_vector_by_direction_to_entity_process`).

Although I believe throwing here makes an easier to understand error message.

Reply:
hm I think I misunderstood, no more comments then from my side, thanks a lot for the cleanup!!!